liquibase-standard/src/main/java/liquibase/database/core/OracleDatabase.java

```diff
@@ -272,6 +272,11 @@ public boolean supportsInitiallyDeferrableColumns() {
         return true;
     }
 
+    @Override
+    public boolean supportsCreateRowDependencies(Class<? extends DatabaseObject> type) {
```

Comment:
Do we need the`type` parameter for this method?  If we do, we should check whether it's a Table type. Otherwise, we can pass any other type here and it will never return `true`.